client/db/types.go

```diff
@@ -176,7 +179,10 @@ type AccountInfo struct {
 	// automatically.
 	LegacyEncKey []byte
 
-	Bonds []*Bond
+	Bonds        []*Bond
+	TargetTier   uint64 // zero means no bond maintenance (allows negative)
```

Comment:
allows negative?

Reply:
`TargetTier` set to 0 allows the users effective trading tier (according to server) to drop into negative.  As in if this is set to zero it's not going to try to keep it at zero but rather let it go.